test/Configuration.spec.js

```diff
@@ -16,8 +21,55 @@ describe("Configuration", function() {
           "Tag: Internal": ":house: Internal"
         },
         "cacheDir": ".changelog",
-        "rootPath": path.resolve(`${__dirname}/..`),
+        rootPath,
+      });
+    });
+  });
+
+  describe("fromPath", function() {
+    const tmpDir = `${os.tmpDir()}/changelog-test`;
+
+    beforeEach(function() {
+      fs.ensureDirSync(tmpDir);
+    });
+
+    afterEach(function() {
+      fs.removeSync(tmpDir);
+    });
+
+    it("reads the configuration from 'lerna.json'", function() {
+      fs.writeJsonSync(path.join(tmpDir, "lerna.json"), {
+        changelog: { repo: "foo/bar" },
+      });
+
+      const result = fromPath(tmpDir);
+      expect(result.repo).toEqual("foo/bar");
+    });
+
+    it("reads the configuration from 'package.json'", function() {
+      fs.writeJsonSync(path.join(tmpDir, "package.json"), {
+        changelog: { repo: "foo/bar" },
+      });
```

Comment:
I've just tried this, but due to the different scenarios that share no code the tests actually seem much harder to read after that refactoring 🤔